test/shim_test/io_test.cpp

```diff
@@ -83,6 +84,22 @@ io_test_init_runlist_cmd(bo* cmd_bo, std::vector<bo*>& cmd_bos)
 }
 
 #define IO_TEST_TIMEOUT 5000 /* millisecond */
+
+void io_test_cmd_wait(hwqueue_handle *hwq, std::shared_ptr<bo> bo)
+{
+    if (io_test_parameters.wait == IO_TEST_POLL_WAIT) {
+        auto start = clk::now();
+        while(!hwq->poll_command(bo->get())) {
+            auto now = clk::now();
```

Comment:
It might be too heavy to call clk::now() for every poll.

Reply:
fixed